maidr/maidr.py

```diff
@@ -10,6 +11,14 @@
 from maidr.core.figure_manager import FigureManager
 
 
+def render(
+    plot: Any, *, lib_prefix: str | None = "lib", include_version: bool = True
+) -> RenderedHTML:
```

Comment:
In order to incorporate the simplified rendering we can return the `Tag` object instead of the `RenderedHTML`.

```
def render(plot: Any) -> Tag:
    ax = FigureManager.get_axes(plot)
    maidr = FigureManager.get_maidr(ax.get_figure())
    return maidr.render()
```

And update core/maidr.py to render the `Tag`
```
    def render(self) -> Tag:
        return self._create_html_tag()
```

Reply:
Thank you @SaaiVenkat, for these suggestions. These changes have been incorporated and have made the module more compact